modules/ROOT/pages/sap/sap-s4-hana-connector.adoc

```diff
@@ -0,0 +1,466 @@
+
+. Select the plus sign to add a component.
+. Select the HTTP connector as a component.
+. Configure these fields:
++
+// image::sap/sap-hana-http-listener.png[] TODO
++
+** Protocol: Protocol selected for the HTTP endpoint, it can be HTTP or HTTPS (secure).
+** Host: IP address where your Mule application listens for requests.
+** Port: Port address where your Mule application listens for requests.
+** Base Path: Path where your Mule application listens for requests.
++
+. Select the plus sign to add a component.
++
+image::sap/sap-hana-plus.png[]
++
+. Select the connector as a component.
+. Select the desired operation.
+. Configure the global element for the connector.
++
+Basic Connection:
++
+// image::sap/sap-hana-conf1.png[] TODO
++
+// image::sap/sap-hana-conf2.png[] TODO
++
+** Service base URL - Enter the URL of the SAP S/4 HANA instance which you wish to manage.
+** Username - Username used to initialize the session.
+** Password - Password used to authenticate the user.
+
+== Studio Configuration
+
+You can use this connector in Anypoint Studio by first downloading it from Exchange
+and configuring it as needed.
+
+=== Install Connector in Studio
+
+. In Anypoint Studio, click the Exchange icon in the Studio taskbar.
+. In Anypoint Exchange, click *Login*.
+. Search for this connector and click *Add to project*.
+. Follow the prompts to install this connector.
+
+Alternatively you can install the connector from your `pom.xml` file:
+
+[source,xml,linenums]
+----
+<dependency>
+    <groupId>com.mulesoft.connectors</groupId>
+    <artifactId>mule-sap-s4hana-cloud-connector</artifactId>
+    <version>LATEST</version>
+    <classifier>mule-plugin</classifier>
+</dependency>
+----
+
+== Configure the SAP S/4 HANA Cloud Connector Global Element
+
+To use the SAP S/4 HANA Cloud connector in a Mule application, you must configure a global SAP S/4 HANA configuration element that can be used by as many SAP S/4 HANA connectors as
+you require for your application.
+
+. Click the *Global Elements* tab at the base of the canvas.
+. In the Global Configuration Elements screen, click *Create*. 
++
+image::sap/sap-hana-global-config-wizard.png[Global Element Configuration Wizard]
++
+. In the Choose Global Type wizard, expand *Connector Configuration*, select *SAP S 4HANA Config*, and click *OK*.
++
+The following configuration screen for Basic Connection appears.
++
+image::sap/sap-hana-basic-config.png[]
++
+. Configure these content:
+
+==== General
+
+[%header%autowidth.spread]
+|===
+|Parameter|Description
+|Service base URL|The base URL of the service on SAP S/4 HANA to be accessed from the connector.
+|Username|User name to connect.
+|Password|Password to connect.
+|===
+
+==== Advanced
+
+[%header%autowidth.spread]
+|===
+|Parameter|Description
+|Service discovery relative path|Relative path from the base URL to the service which will be used for automatic services discovery.
```

Comment:
Service discovery relative path|Relative path from the base URL to the service to use for automatic services discovery.